npc/custom/quests/valhallen.txt

```diff
@@ -19,21 +24,21 @@ prontera,147,171,5	script	Lenneth	811,{
 	mes "I can transmute for you many of the Valhallen items,";
 	mes "although they require many rare elements to make.";
 	next;
-	switch(select("Mjolnir:Sleipnir:Brisingamen:Gleipnir:Megingjard")) {
+	switch(select("Mjolnir:Sleipnir:Brisingamen:Gleipnir:Megingjard:Asprika:Brynhild:Hervor:Jormungand")) {
 		case 1:
 			mes "[Lenneth]";
 			mes "^3355FFMjolnir^000000 is the mighty Hammer of Thor.";
-			setarray .@Items[0],1530,1531,2,984,20,985,5,969,40,7074,2,7075,4,7078,5,7087,5,7089,5;
+			setarray .@Items[0],1530,1522,2,984,20,985,5,969,40,7074,2,7075,4,7078,5,7091,5,7089,5;
 			break;
 		case 2:
 			mes "[Lenneth]";
 			mes "^3355FFSleipnir^000000 are boots made after Odin's War Horse.";
-			setarray .@Items[0],2410,2406,2,984,1,969,20,985,10,7076,3,7079,5,7083,3,7086,3;
+			setarray .@Items[0],2410,2406,2,984,20,969,20,985,10,7076,3,7079,5,7083,3,7086,4,7087,3;
```

Comment:
From https://github.com/rathena/rathena/blob/master/npc/quests/seals/god_weapon_creation.txt#L650
and IRO Sleipnir doesn't require 20 Oridecon (**ID: 984**)